src/schema/auth-request-body.ts

```diff
@@ -0,0 +1,16 @@
+import { JSONSchemaType } from 'ajv'
+import { AuthRequestBody } from '../types'
+
+export const authRequestBodySchema: JSONSchemaType<AuthRequestBody> = {
+  $id: 'AuthRequestBodySchema',
+  type: 'object',
+  properties: {
+    message: {
+      type: 'string',
+    },
+    signature: {
```

Comment:
Right now the spec requires that `signature` be a JSON. I understand why it's a string here though, so I'm happy to change it in the spec. Just so I'm clear, the signature is exactly 130 hex characters (65 bytes) prefixed by `0x`?

Reply:
yep that's correct. The first 32 bytes is `r`, next 32 bytes is `s` and the final byte is `v`.